.github/workflows/docker.yml

```diff
@@ -690,7 +690,7 @@ jobs:
           repo: netdata/helmchart
           workflow: Agent Version PR
           ref: refs/heads/master
-          inputs: '{"agent_version": "${{ needs.gen-tags.outputs.tags }}"}'
+          inputs: '{"agent_version": "${{ inputs.version }}"}'
```

Comment:
I think this should be `github.event.inputs.version`, right?

Reply:
It used to require that, but at some point within the past few months GitHub decided to add a new top-level context for this called `inputs`. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs.

The old `github.events.inputs` path also works, but AFAICT it’s for backwards compatibility only, and it also makes things needlessly longer, so I would tend to prefer the newer `inputs` context for any new code.